components/kyma-metrics-collector/pkg/keb/client.go

```diff
@@ -108,13 +114,17 @@ func (c Client) GetRuntimesPerPage(req *http.Request, pageNum int) (*kebruntime.
 	}
 
 	if resp.StatusCode != http.StatusOK {
+		totalRequest.WithLabelValues(fmt.Sprintf("%d", resp.StatusCode)).Inc()
```

Comment:
we only need to do this once, so if you move this to line 115, all other similar calls in this function can be removed

Reply:
done